editors/sc-ide/widgets/code_editor/highlighter.hpp

```diff
@@ -89,7 +89,7 @@ class SyntaxHighlighter : public QSyntaxHighlighter {
     SyntaxHighlighter(QTextDocument* parent = 0);
 
 private:
-    void highlightBlock(const QString& text);
+    void highlightBlock(const QString& text) override;
```

Comment:
I'm curious - why is the override keyword needed now and not before? Or - was part of the problem that this wasn't marked as override properly?

Reply:
it wasn't there before, this is very much a "leave the code better than you found it" commit